src/components/Slide/index.tsx

```diff
@@ -27,8 +30,9 @@ const Slide = ({ movies }: SlideProps) => {
     setCurrentSlide(currentSlide + direction);
   };
 
+  if (!movies.length) return pathname === '/' ? null : <NoData />;
```

Comment:
null이 있으니 뭔가 이상해보이네요 아래처럼 수정할 수 있지않을까 하여 코멘트 남깁니다!
```suggestion
  if (!movies.length && pathname === '/') return <NoData />;
```